packages/eslint-plugin/package.json

```
@@ -50,15 +50,15 @@
   ],
   "scripts": {
     "build": "tsc -b tsconfig.build.json",
-    "clean": "tsc -b tsconfig.build.json --clean",
-    "postclean": "rimraf dist && rimraf coverage",
-    "format": "prettier --write \"./**/*.{ts,mts,cts,tsx,js,mjs,cjs,jsx,json,md,css}\" --ignore-path ../../.prettierignore",
+    "clean": "tsc -b tsconfig.build.json --clean & yarn run postclean",
+    "postclean": "rimraf dist/ coverage/",
+    "format": "yarn run -T format",
     "generate:breaking-changes": "tsx tools/generate-breaking-changes.mts",
-    "generate:configs": "npx nx generate-configs repo",
-    "lint": "npx nx lint",
+    "generate:configs": "yarn run -T generate-configs",
```

It might be worth doing something like this at some point in order to prevent accidentally creating [`global-scripts`](https://yarnpkg.com/features/workspaces#global-scripts)

```suggestion
    "generate-configs": "yarn run -T generate-configs",
```